### ISSUE TITLE
[Bug] [BD] Unread channels wrong scroll-up amount

### ISSUE THREAD

**Issue by Davoleo** [Saltssaumure/ieytd-discord-theme#13]
BD Version: `1.10.1`
Theme Version: `2.3.1`

**Description**: When clicking on a server channel with unread messages the channel view is automatically scrolled up a wrong amount of messages:
- if unread messages are < than ~30 chat window will be scrolled up more than needed and you'll need to scroll down a certain amount to reach newer messages
- if unread messages are > than ~30 chat window will be scrolled up partially forcing you to scroll up even more to reach the oldest unread message
  - while I know that even vanilla discord usually doesn't scroll up more than 100 messages if there are so many unread ones, this issue is still present even for amounts that sit between 30 and 100
- Additionally, idk if this is just me being blind, but I wasn't able to see the "unread messages start here" line that I usually see on discord that I use to check what message I need to read first, making the issue described above even more frustrating

I don't know if this is the case but could it be related to che changes in size on many components in this theme?

**Comment by Saltssaumure** [comment]
- Can't seem to replicate the scroll issue at 100% zoom and 16px chat font size. Are you using different settings for either of those?
- Will make unread line visible in next version.

**Comment by Davoleo** [comment]
1. mmh, this is weird, it occurs to me even after disabling any other plugin/theme I had enabled, no idea why it might be happening. 
I'm currently using 100% zoom and 16px font size too, changes to those don't seem to affect the issue though, as far as I see, maybe they make it more prominent if changed at all. 
I also noticed that this issue doesn't happens on threads for some reasons, although I could only check 1 time, there was only 1 new message and I didn't have to scroll at all to reach it unlike on normal channels.
2. Thanks! that will definitely help

Edit: I'll add that I'm on a 1920x1080 monitor with 100% Scale and Layout (Windows 10)
